dune-project

```diff
@@ -10,50 +11,38 @@
  (name solver-service)
  (synopsis "Choose package versions to test")
  (depends
-  (ocaml (>= 4.14.1))
-  (alcotest-lwt (and (>= 1.7.0) :with-test))
+  (ocaml (>= 5.0.0))
   ; Examples dependencies
   (current_web (and (>= 0.6.4) :with-test))
   (current_git (and (>= 0.6.4) :with-test))
   (current_github (and (>= 0.6.4) :with-test))
   (current_ocluster (and (>= 0.2.1) :with-test))
-  (ocluster-api (and (>= 0.2.1) :with-test))
   (ppx_deriving_yojson (>= 3.6.1))
   (ppx_deriving (>= 5.1))
   (yojson (>= 2.1.0))
   (lwt (>= 5.6.1))
+  (eio (>= 0.11))
+  (eio_main (>= 0.11))
+  (lwt_eio (>= 0.4))
   (logs (>= 0.7.0))
   (fmt (>= 0.9.0))
   (ocaml-version (>= 3.6.1))
   (solver-service-api (= :version))
   (dune-build-info (>= 3.8.0))
-  conf-libev
   (opam-0install (>= 0.4.3))
   (git-unix (>= 3.12.0))
+  (ocluster-api (>= 0.2.1))
+  (prometheus-app (>= 1.2))
```

Comment:
Is prometheus threadsafe?

Reply:
No, but it's only used from the main domain.